filetracker/servers/migration.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import email.utils
+import os.path
+from hmac import new
+from uu import encode
+
+from six.moves.urllib.parse import parse_qs
+
+from filetracker.servers import base
+from filetracker.servers.files import FileTrackerServer
+from filetracker.servers.storage import FileStorage
+
+
+class MigrationFileTrackerServer(FileTrackerServer):
+    """A WSGI application providing a HTTP server compatible with
+       :class:`filetracker.RemoteDataStore`
+       that redirects GET requests for missing files to another server."""
+
+    def __init__(self, redirect_url, dir=None):
```

Comment:
Nitpick: dir should be the first parameter, since it's passed to the superclass.

Reply:
It's not possible, because dir is optional, and redirect_url is required. And all required parameters must come before optional ones.